aws/ci_build.rake

```diff
@@ -138,16 +138,6 @@ task :apps_task do
   packager.decompress_package(package)
 end
 
-# TODO: This is a temporary task meant to cleanup old code-studio packages.
-# It should go away in the nearish future
-task :code_studio_task do
-  HipChat.log 'Cleaning code-studio package'
-  # get rid of any symlink to a built package
-  RakeUtils.system 'rm', dashboard_dir('public/code-studio') if File.exist?(dashboard_dir('public/code-studio'))
-  # also delete the package itself
-  RakeUtils.system 'rm -rf', dashboard_dir('public/code-studio-package') if File.exist?(dashboard_dir('public/code-studio-package'))
-end
-
```

Comment:
If it is baked into the image, we should update the starting image. Any idea how to go about doing so?